### PR TITLE
add conditional to not build for el6 (software-4028)

### PR DESCRIPTION
**Pull request by ddavila0**

### CODE REVIEW

**Review by djw8605 — COMMENTED** [review]
@matyasselmeci do you know, would this every pass an EPEL packaging review?  That's the eventual goal of this package.

**Comment by matyasselmeci** [comment]
I doubt it. Thinking about it again, perhaps a better solution would be
```
# Only on EL 7 and newer:
Requires: gcc-c++ >= 4.8
```

File: rpm/xrootd-scitokens.spec [hunk 1]
```diff
@@ -9,6 +9,9 @@ URL: https://github.com/scitokens/xrootd-scitokens
 # git archive v%{version} --prefix=xrootd-scitokens-%{version}/ | gzip -7 > ~/rpmbuild/SOURCES/xrootd-scitokens-%{version}.tar.gz
 Source0: %{name}-%{version}.tar.gz
 
+# Only on EL 7 and newer:
+Requires: gcc-c++ >= 4.8
```
Comment (matyasselmeci) [inline rpm/xrootd-scitokens.spec]:
```suggestion
BuildRequires: gcc-c++ >= 4.8
```
My bad, I should have said this in the first place.

Comment (matyasselmeci) [reply rpm/xrootd-scitokens.spec]: Can you apply this change? Then we can merge it.

**Review by matyasselmeci — APPROVED** [review]

**Comment by matyasselmeci** [comment]
@djw8605 can you merge this? I don't have the privs.

**Pull request merged**